libpod/runtime_ctr.go

```diff
@@ -195,6 +204,49 @@ func (r *Runtime) GetContainers(filters ...ContainerFilter) ([]*Container, error
 	return ctrsFiltered, nil
 }
 
+// LogOptions is used for the GetLogs function
+// holds flag params defined by the user
+type LogOptions struct {
+	Details   bool
+	Follow    bool
+	SinceTime time.Time
+	Tail      uint64
+}
+
+// GetLogs returns the logs of a container from the log file
+// log file is created when the container is started/ran
+func (r *Runtime) GetLogs(container string, logChan chan string, opts LogOptions) error {
```

Comment:
Can we move this out to cmd/kpod? I don't think we need anything in the runtime for this aside from the container, which you can just get with LookupContainer

Reply:
fixed.